vignettes/using-delayed-data-basic.Rmd

```diff
@@ -82,7 +103,17 @@ Notes:
 The datasets passed into `teal_data` and `cdisc_data` are pulled in the order they are inputted. So if datasets depend on other datasets being available they should be placed later in the argument list:
 
 ```{r}
-adsl <- scda_cdisc_dataset_connector(dataname = "ADSL", "adsl")
+# nolint start
+library(teal.data)
+library(magrittr)
+
+adsl <- cdisc_dataset_connector(
+  dataname = "ADSL",
+  pull_callable = callable_function(function() example_cdisc_data("ADSL")),
```

Comment:
This connector is badly specified. 
```
adsl$get_code()
# "ADSL <- (function() example_cdisc_data(\"ADSL\"))()"
```

Connector should be specified with some function and set_args
```r
adsl <- cdisc_dataset_connector(
  dataname = "ADSL",
  pull_callable = callable_function(teal.data::example_cdisc_data)$set_args(list(dataname = "ADSL")),
  keys = get_cdisc_keys("ADSL"),
  label = "ADSL connector"
)
adsl$get_code()
"ADSL <- teal.data::example_cdisc_data(dataname = \"ADSL\")"
```

Reply:
Yes , this is updated. 
some places I have used with magrittr  pipe operator
```
callable_function(example_cdisc_data) %>%
  set_args(list(dataname = "<dataname>").
```